src/ZEO/tests/test_marshal.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from ZEO.asyncio.marshal import encode
+from ZEO.asyncio.marshal import pickle_server_decode
+from ZopeUndo.Prefix import Prefix
```

Comment:
Can we make the import "soft", and drop the hard testing dependency?  E.g.:

```python
try:
    from ZopeUndo.Prefix import Prefix
except ImportError:
    _HAVE_ZOPE_UNDO = False
else:
    _HAVE_ZOPE_UNDO = True
```

and then:

```python
    @unittest.skipUnless(_HAVE_ZOPE_UNDO)
    def testServerDecodeZopeUndoFilter(self):
        ...
```

Reply:
FTR: In Plone, we use the other approach checking for the package with ``pkg_resources``as documented in the [Style Guide under # 4](https://docs.plone.org/develop/styleguide/python.html#about-imports). I am not sure if this is needed in this case.